l10n_pa_reports/model/account_invoice.py

```diff
@@ -2,28 +2,12 @@
 # Copyright 2016 Vauxoo (https://www.vauxoo.com) <info@vauxoo.com>
 from __future__ import division
 import time
-from openerp import models, api, fields
+from openerp import models, api
 
 
 class AccountInvoice(models.Model):
     _inherit = 'account.invoice'
 
-    l10n_pa_concept = fields.Selection([
-        (1, 'Compras o Adquisiciones de Bienes Muebles'),
-        (2, 'Servicios Básicos'),
-        (3, 'Honorarios y Comisiones por Servicios'),
-        (4, 'Alquileres por Arrendamientos Comerciales'),
-        (5, 'Cargos Bancarios, Intereses y Otros Gastos Financieros'),
-        (6, 'Compras o Servicios del Exterior'),
-        (7, 'Compras o Servicios Consolidados')],
-        string='Supplier Invoice Concept',
-        help='Indicates the Concept of the Supplier Invoice.')
-    l10n_pa_prd_srv = fields.Selection([
```

Comment:
@hbto I'm still getting this error on instance with legacy data, the instance always updates with the latest from all repos. As you can see in the top right corner this one is from this morning:

![a](http://screenshots.vauxoo.com/tulio/10342229316-BUybgNAeYj.jpg)

And is using the latest database from yoytec-legacy build in runbot2, ATM I have to reset (manually) to the previous commit to get it fixed, so far is the only thing that have worked for me not even updating a bunch of times